lib/date_picker.dart

```diff
@@ -189,6 +191,8 @@ class DatePicker {
             _selectedDate = date;
           }),
           looping: looping,
+          dividerHeight: dividerHeight,
```

Comment:
I would suggest to move this to the theme instead, (Since the dividerColor is passed there)